hack/scaletest/README.md

```diff
@@ -4,29 +4,114 @@ Simulate 1000 clusters in Hive and observe performance. Because we do not have t
 
 # Setup
 
-Hive running on a standard 3 master 3 node OpenShift 4.2 cluster.
+Hive running on a standard 3 master 3 node OpenShift 4.4 cluster. (instance types for workers changed later)
 
-Another identical cluster is being used for the duplication.
+Create 60 SelectorSyncSets, each will contain 10 unique ConfigMaps with a little data, in the default namespace:
 
-```
-$ time hack/scaletest/test_setup.sh cluster.kubeconfig 1000
+```bash
+hack/scaletest/setup-selectorsyncsets.sh
 ```
 
-Once we have 1000 clusters, create 30 SelectorSyncSets with different resources, which match all our clusters. (repeat for 0-29)
+Provision a real spoke cluster from the Hive hub cluster:
 
+```bash
+bin/hiveutil create-cluster --base-domain=new-installer.openshift.com --use-image-set=false scaletest-spoke
 ```
-oc process -f hack/scaletest/syncset-template.yaml RESOURCE_INDEX=0 | oc apply -f -
+
+Get the cluster's admin kubeconfig which will be used to adopt this one cluster into thousands of duplicates, all backed by the same real spoke cluster:
+
+```bash
+./hack/get-kubeconfig.sh scaletest-spoke > scaletest-spoke.kubeconfig
 ```
 
-Setup a transient prometheus pod in the hive namespace (see developer docs) and used the following queries to monitor Hive behavior:
+Duplicate the real cluster by adopting copies to simulate cluster load in Hive. This can be run in multiple terminals to parallelize and generate more load, just use different start/end blocks in each.
```

Comment:
you can do the same by running the test_setup.sh script in the background, like:
```
$ time hack/scaletest/test_setup.sh cluster.kubeconfig 1 100 &
$ time hack/scaletest/test_setup.sh cluster.kubeconfig 101 200 &
...
```